SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -102,7 +102,8 @@ mount_overlayfs() {
     else
         echo "Mounting regular root"
         mkdir -p "${VERITY_MOUNT}"
-        mount -o ro,defaults "$root" "${VERITY_MOUNT}" || \
+        root_device="${root#block:}"  # Remove 'block:' prefix if present.
```

Comment:
curious, what's scenario where block will be present?

Reply:
When `getarg` consume `$root=` from the kernel args, it will return something for example `block:/dev/disk/by-partuuid/301574d7-db66-4f13-9492-6208859a1b27`.

Reply:
I'm not sure if this is still the standard way to handle root paths, but consider https://github.com/microsoft/azurelinux/blob/592875ba994cf7fc55449f95bfcb3d26f618a50d/SPECS/verity-read-only-root/20verity-mount/verity-parse.sh#L111

Reply:
Fixed, please review again, thanks.